### DISCUSSION TITLE
Abrindo ME fator R

### DISCUSSION THREAD

**Discussion by YShow** [DyegoCosta/trabalhando-remoto#167] [General]
Fala pessoal, recentemente tiver que trocar de MEI para ME devido ao valor que recebeo, decidi por abrir outro CNPJ pela rapidez e meu cliente querendo que troque, fiquei com uma duvida que nao achei em qualquer lugar e meu contador parece confuso.
Para eu poder calcular o Fator R eu devo ter a minha primeira renda como Anexo V e somento a partir do segundo mes tenho como fazer parte do Anexo III?
Exemplo: Abri minha empresa hoje 12/05, meu primeiro faturamento nessa nova empresa sera no proximo mes, dia 01-05/06/22, ja consigo calcular o fator R a partir do primeiro faturamento ou devo pagar o Anexo V e comecar a calcular a partir do proximo mes?

**Comment by pablo-sg-pacheco** [comment]
Eu não sou especialista, mas acho que o fator R só começa a calcular depois do primeiro ano.

**Comment by YShow** [comment]
Opa Pablo, e possivel calcular o fator R ja no primeiro ano, mas de acordo com meu contador o primeiro mes ainda fico no Anexo V e somente depois consigo Anexo III

**Comment by pablo-sg-pacheco** [comment]
Eu fiquei como anexo 5 durante uns bons meses até que eu conseguisse ir pro Anexo 3.

Se vc se pagar um pró-labore com um valor de até 28% do seu faturamento, no meu entender vc já poderia ficar no Anexo 3 desde o começo. 

Eu acabei tendo que me pagar bem mais do que isso pra ir compensando.

**Comment by copercini** [comment]
Comigo ficou o primeiro mês no Anexo V e depois foi pro anexo III usando as médias dos meses anteriores para o cálculo do fator R

**Comment by heliomeiralins** [comment]
Como seu primeiro faturamento é só no próximo mês, vc pode retirar um pró-labore esse mês de um salário mínimo e já entra no fator R no mês seguinte.

**Comment by YShow** [comment]
> Comigo ficou o primeiro mês no Anexo V e depois foi pro anexo III usando as médias dos meses anteriores para o cálculo do fator R

Entendi, bom vo confirmar com meu contador novamente, vlw pela ajuda

**Comment by YShow** [comment]
> Como seu primeiro faturamento é só no próximo mês, vc pode retirar um pró-labore esse mês de um salário mínimo e já entra no fator R no mês seguinte.

Acho que e isso que meu contado vai fazer entao, infelizmente vou entrar esse mes no Anexo V :(

**Comment by igorlvicente** [comment]
Aqui também foi o primeiro mês normal e a partir do segundo que entrei no fator R.
Se você perguntou só pra confirmar o que seu contador disse, tudo bem, mas se seu contador não soube te instruir sobre isso, recomendo procurar outra contabilidade.

**Comment by YShow** [comment]
> Aqui também foi o primeiro mês normal e a partir do segundo que entrei no fator R. Se você perguntou só pra confirmar o que seu contador disse, tudo bem, mas se seu contador não soube te instruir sobre isso, recomendo procurar outra contabilidade.

Opa Igor, era so pra confirmar e ver se algo nao tinha mudado, o contador e bom e uma pessoa de confianca, era so pra ver se nao tinha como se encaixar de alguma forma.

Vlw pela ajuda pessoal
